packages/nextjs/hooks/scaffold-stark/useScaffoldMultiWriteContract.ts

```diff
@@ -9,10 +9,18 @@ import {
   UseScaffoldWriteConfig,
 } from "~~/utils/scaffold-stark/contract";
 import { useSendTransaction, useNetwork, Abi } from "@starknet-react/core";
-import { Contract as StarknetJsContract, InvocationsDetails } from "starknet";
+import {
+  Contract as StarknetJsContract,
+  InvocationsDetails,
+  Call,
+} from "starknet";
 import { notification } from "~~/utils/scaffold-stark";
 import { useTransactor } from "./useTransactor";
 
+function isRawCall(value: Call | any): value is Call {
+  return "entrypoint" in value;
```

Comment:
interesting what is this syntax ser? @iossocket

Reply:
verify the type is `Call` or `UseScaffoldWriteConfig<TAbi, TContractName, TFunctionName>`, only `Call` has the `entrypoint` property